site/gatsby-ssr.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { getCriticalHDSRules, hdsStyles } from 'hds-react';
+import { renderToString } from 'react-dom/server';
+
+export const replaceRenderer = async ({ bodyComponent, setHeadComponents }) => {
+  const bodyHTML = renderToString(bodyComponent);
+
+  if (hdsStyles && hdsStyles.length > 0 && bodyHTML && bodyHTML.length > 0) {
+    const cssRules = await getCriticalHDSRules(bodyHTML, hdsStyles);
+    const HeadComponents = [
+      <style data-used-styles dangerouslySetInnerHTML={{ __html: cssRules }} />
```

Comment:
Maybe we could do this inside the Seo component instead of this gatsby-ssr.js file?